src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,75 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pages = getNumbers(1, Math.ceil(total / perPage));
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', {
+        disabled: pages.indexOf(currentPage) === 0,
+      })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={pages.indexOf(currentPage) === 0}
+          onClick={() => {
+            if (pages.indexOf(currentPage) !== 0) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+        >
+          «
+        </a>
+      </li>
+
+      {pages.map(page => (
+        <li key={page} className={`page-item ${currentPage === page && 'active'}`}>
```

Comment:
use classnames package to avoid such cases
<img width="267" alt="image" src="https://github.com/mate-academy/react_pagination/assets/70220667/b6a5083a-6208-4f01-86f6-24e1a227d8f0">